tests/units/anta_tests/__init__.py

```diff
@@ -72,7 +69,9 @@ def test(device: AntaDevice, anta_test: type[AntaTest], unit_test_data: AntaUnit
             f"Expected {len(unit_test_data['expected']['messages'])} messages, got {len(test_instance.result.messages)}"
         )
         # Test will pass if the expected message is included in the test result message
-        for message, expected in zip(test_instance.result.messages, unit_test_data["expected"]["messages"]):  # NOTE: zip(strict=True) has been added in Python 3.10
+        for message, expected in zip(
+            test_instance.result.messages, unit_test_data["expected"]["messages"], strict=False
+        ):  # NOTE: zip(strict=True) has been added in Python 3.10
```

Comment:
Should we add strict=True here? Otherwise the comment can be removed.
```suggestion
            test_instance.result.messages, unit_test_data["expected"]["messages"], strict=False
        ):
```